change/@fluentui-react-slider-03252d64-5321-44f5-a760-9bbdadf077a3.json

```
@@ -0,0 +1,7 @@
+{
+  "type": "prerelease",
+  "comment": "Refactor slider component",
```

```suggestion
  "comment": "Refactor slider component: removed marks support, RangedSlider is removed",
```

Let's make changelog entry more useful